File: benchmark/calc.js
const helpers = require('../utils/helpers')

module.exports = function () {
  const beg = helpers.now()
  let sum = 0
  for (let i = 0; i < 1000000000; i++) {
    sum += 1000000000.123 * 1000000000.321 * 321.123456789
  }
  const end = helpers.now()

  helpers.log(`Nodejs : Calc used ${end - beg} ms`)
}

File: benchmark/calc.go
package benchmark

import "github.com/joway/golang-talk/utils"

func Calc() {
	beg := utils.Now()
	sum := 0.0
	for i := 0; i < 1000000000; i++ {
		sum += 1000000000.123 * 1000000000.321 * 321.123456789
	}
	end := utils.Now()
	utils.Logf("Go : Calc used %d ms\n", end-beg)
}
